napari_skimage_regionprops/_parametric_images.py

```diff
@@ -1,7 +1,87 @@
 import numpy as np
 from napari_tools_menu import register_function
 import numpy
+from deprecated import deprecated
+from napari.utils import notifications
```

Comment:
Can you please delay this import? (Import it from within the function where you use it)

Reply:
Sure! Good that you remember me, thanks!